templates/documentpage.html

```diff
@@ -0,0 +1,23 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>Mariapp</title>
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
These imports should be cleaned up in a future commit, when we design the api calls and such.

Reply:
For these imports, I'll leave them like this for now because I don't know how to clean them up. I'll let you or Laurence handle it.